knexfile.js

```diff
@@ -5,7 +5,11 @@ const env = require('dotenv').config();
 let dotenvExpand = require('dotenv-expand');
 dotenvExpand(env);
 
-const BASE_PATH = path.join(__dirname, 'src', 'wallet', 'db');
+const sourceDirectory = 'src';
+if (process.env.NODE_ENV === 'production') {
```

Comment:
This is a bit pedantic, but I think we should call the current heroku environment staging for now?

Reply:
Agreed that we need a staging environment. Currently, our Heroku app is a named `rps-autoplayer`. If we want to use this app as the staging environment, we should probably rename it to `rps-autoplayer-staging`. The other alternative, is to use the the current app for production and spin up a new staging app.